packages/design-system/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/react-dom": "^17.0.10",
     "classnames": "^2.2.5",
     "core-js": "^3.6.5",
+    "dialog-polyfill": "^0.5.6",
```

Comment:
Needed to support IE and older versions of Safari and Firefox

Reply:
Actually I think Patrick has a branch up addressing a `window not defined bug` that might need merged here

Reply:
If you've looked at those changes and like them, can you merge them or cherry-pick them into this branch?

Reply:
Will do!